los-deferal-02/Hello-Books
# #166238090 Build Email Verification Service

#### What does this PR do?

- Add email verification service for new users when they sign up.

#### Description of Task to be completed?

- Add email confirmation column to database users table

- Send new users email when they register with link for verifying their email

- Add email verification endpoint 

- Write tests to cover feature added

#### How should this be manually tested?

- Clone the repo and run `npm run test` from the server folder

- From Postman, make a fresh sign up request. Open the email received on registration and click on the verify email link.

- Make a patch request using this URL on Postman and a message confirming the email has been verified will be received

#### Any background context you want to provide?
This feature enables the app administrators to verify the emails of users when they register. This will ensure that users only register with emails they have access to.

#### What are the relevant pivotal tracker stories?
[#166238090](https://www.pivotaltracker.com/story/show/166238095)

#### Screenshots (if appropriate)
![Screenshot (107)](https://user-images.githubusercontent.com/42539315/58907755-cec3c200-8706-11e9-8ba4-c213ba5f4a4d.png)


![Screenshot (106)](https://user-images.githubusercontent.com/42539315/58907664-9fad5080-8706-11e9-9761-1107580471cb.png)
